alerting/src/test/kotlin/org/opensearch/alerting/settings/AlertingSettingsIT.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.settings
+
+import org.opensearch.alerting.ALWAYS_RUN
+import org.opensearch.alerting.AlertingRestTestCase
+import org.opensearch.alerting.model.Trigger
+import org.opensearch.alerting.model.action.Action
+import org.opensearch.alerting.randomAction
+import org.opensearch.alerting.randomQueryLevelMonitor
+import org.opensearch.alerting.randomQueryLevelTrigger
+
+class AlertingSettingsIT : AlertingRestTestCase() {
+
+    fun `test updating setting of max actions per trigger with less actions than allowed actions`() {
+        val actions = createActions(4)
+        val triggers = createTriggers(1, actions)
+        createMonitor(
+            randomQueryLevelMonitor(
+                triggers = triggers
+            )
+        )
+
+        try {
+            client().updateSettings(AlertingSettings.TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS.key, 1)
```

Comment:
this should be `TOTAL_MAX_ACTIONS_PER_TRIGGER` since the test says we are testing per trigger